My portfolio

eerapheal opened the pull request:
These are two sections of the mobile version of my portolio Build with Html and CSS programing language.

eerapheal commented:
I change the previous font family and also add alt to html.

Bondok6 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+    <section>
+      <div class="headline-div">
+        <h1 class="headerline">Hey There. I’m Ralphael</h1>
+        <p class="sub-headerline">
+          I’am a Software Developer
+        </p>
```
- [ ] Great job using the semantic HTML5 tags correctly 💯👌, but I think here it would be better using a header tag `hx` (examples: `<h2>`,`<h3>`, etc...) instead of  `<p>` tag. Because it is considered a subtitle.

eerapheal replied:
I have changed it to <h2>

Bondok6 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+  <main>
+    <section>
+      <div class="headline-div">
+        <h1 class="headerline">Hey There. I’m Ralphael</h1>
+        <p class="sub-headerline">
+          I’am a Software Developer
+        </p>
+         <p class="content-p">
+           I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
```
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, change your font color to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/176821196-82609213-990f-4a54-a234-3cab3236468e.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/176821124-f6360851-fc31-455c-838b-c79765cd0186.png" width="500" /> |

eerapheal replied:
I have changed the font color to #FF6B00

Bondok6 commented on index.html:
```diff
@@ -0,0 +1,40 @@
+         <p class="content-p">
+           I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div id="social-media">
+          <a href="#" target="_blank"> <img width="20" height="25" src="image/githubimage/V.png" alt="icon"></a>
+          <a href="#" target="_blank"> <img width="20" height="25" src="image/V.png" alt="icon"></a>
+          <a href="#" target="_blank"> <img width="20" height="25" src="image/twitter" alt="icon"></a>
+          <a href="#" target="_blank"> <img width="20" height="25" src="image/me" alt="icon"></a>
+        </div>
```
- [ ] the list of icons should be grouped by a `<ul>` tag and each `<a>` tag should be in `<li>` tag, kindly fix that. 👍 
> For example
```
<ul id="social-media">
  <li><a href="#" target="_blank"> <img width="20" height="25" src="image/githubimage/V.png" alt="icon"></a></li>
  <li><a href="#" target="_blank"> <img width="20" height="25" src="image/V.png" alt="icon"></a></li>
  <li><a href="#" target="_blank"> <img width="20" height="25" src="image/twitter" alt="icon"></a></li>
  <li><a href="#" target="_blank"> <img width="20" height="25" src="image/me" alt="icon"></a></li>
</ul>
```

- [ ] Aslo, I noticed that the first icon link is broken [GitHub Icon], kindly fix it 👌
![image](https://user-images.githubusercontent.com/45231276/176822354-fa86e8a1-726b-4883-a9f1-32568c50603e.png)

eerapheal replied:
I have fixed the <ul> and <li> on the icon link list and also fix the broken Github icon.

Bondok6 commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  box-sizing: border-box;
+}
+
+body {
+  width: 375px;
+  background-color: lightgray;
+}
```
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, change your `background-color`  to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

eerapheal replied:
I have changed the color to Figma Design.

Bondok6 reviewed (changes requested):
## Hi @eerapheal 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media3.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Highlights
- You've used the correct Github flow ✅
- All linters are passing ✅
- Good PR description ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eerapheal commented:
- I have changed it to <h2>
- I have changed the font color to #FF6B00
- I have fixed the <ul> and <li> on the icon link list and also fix the broken Github icon.
- I have changed the color to Figma Design.

cwaku reviewed (changes requested):
Hi @eerapheal

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- [x] Great overall template design 👍🏾 
- [ ] Good PR description 💯 
- [ ] Correct use of Github flow 🥇 


## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions
[OPTIONAL]
- [ ] I strongly suggest that write meaningful commit messages. Instead of numbering as you have done, Kindly give a short message of the changes you did in a particular commit.

Not so meaningful | Meaningful
-- | --
![image](https://user-images.githubusercontent.com/29071984/176842653-acf20ef4-597e-40f3-b890-91a178a594ca.png) |  ![image](https://user-images.githubusercontent.com/29071984/176842825-4c73675e-2590-4909-a5a1-2b1fe5137d4d.png)


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eerapheal commented on index.html:
```diff
@@ -0,0 +1,42 @@
+    <section>
+      <div class="headline-div">
+        <h2 class="headerline">Hey There. I’m Ralphael</h2>
+        <p class="sub-headerline">
+          I’am a Software Developer
+        </p>
```
okay, changed.

eerapheal commented:
All required fixed.

vickymarz commented on index.html:
```diff
@@ -0,0 +1,38 @@
+        <div class="headline-div">
+          <h1 class="headerline">Hey There. I’m Ralphael</h1>
+          <h2 class="sub-headerline">I’am a Software Developer</h2>
+          <p class="content-p">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
[OPTIONAL]

- [ ] You have done great on your design 📴 . Kindly ensure that the text description matches the one in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27). I think you should work on the font size, font weight, and the line-height 🥳 . 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/176943152-67dd509d-a711-4c24-a7b0-c4a580945632.png)  | ![image](https://user-images.githubusercontent.com/63473584/176943334-99ab689e-a896-46a7-9c64-d27ff377e75b.png)  |

vickymarz reviewed (approved):
Hi @eerapheal , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

eerapheal commented:
I just completed section two of my portfolio website. all checks were done as well.

Kossi-stack commented:
Stay online on slack

Kossi-stack commented:
Whenever you submit for PR
